app/src/main/java/swen_anigans/mathematicfanatic/QuizActivity.java

```diff
@@ -42,22 +42,70 @@ protected void onCreate(Bundle savedInstanceState) {
             initializeQuizQuestions();
 
             for (int i = 0; i < totalPages; i++) {
-                answers.set(i, 0);
-                expectedAnswers.set(i, -1);
+                answers.add(0);
+                expectedAnswers.add(-1);
             }
         }
 
+        /*

```

Comment:
Is this commented out code the same from the bottom? and why is it commented it out. Same for the commented code below.